### ISSUE TITLE
contractile ring multi-layered architecture

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#14337]
I''m curating a paper about contractile ring architecture
Nanoscale architecture of the Schizosaccharomyces pombe contractile ring

![cont](https://user-images.githubusercontent.com/7359272/31392751-f9939600-add1-11e7-9784-0d943fb8071e.jpg)

do you think  that terms for the  

proximal layer and the intermediate layer (accessory protein network)

are appropriate?

we already have
GO:1903144    actomyosin contractile ring actin filament
(maybe actin filament layer would be better?)

**Comment by krchristie** [comment]
McDonald NA, Lind AL, Smith SE, Li R, Gould KL. Nanoscale architecture of the 
Schizosaccharomyces pombe contractile ring. Elife. 2017 Sep 15;6. pii: e28865.
doi: 10.7554/eLife.28865.  PMID: 28914606

**Comment by krchristie** [comment]
@ValWood Do you think that these layers are generally applicable, or pombe specific? If you think they are general, then it definitely seems appropriate to add some more terms for the layers of the contractile ring. If you think this is relatively specific to pombe, let me know if you think these are terms you need to annotate, and more specifics about how to name and define them.

Regarding GO:1903144 actomyosin contractile ring actin filament - this term is defined as a type of actin filament. It seems it might be better to leave this existing term as what it is. If you think we need terms for the three layers, then the existing term can be part_of the contractile ring distal layer (e.g. contractile ring actin filament layer).

**Comment by krchristie** [comment]
Here are some review that came up in a quick PubMed search:

1: Pollard TD. Nine unanswered questions about cytokinesis. J Cell Biol. 2017 Oct
2;216(10):3007-3016. doi: 10.1083/jcb.201612068. Epub 2017 Aug 14. Review. 
PMID: 28807993

2: Thieleke-Matos C, Osório DS, Carvalho AX, Morais-de-Sá E. Emerging Mechanisms 
and Roles for Asymmetric Cytokinesis. Int Rev Cell Mol Biol. 2017;332:297-345.
doi: 10.1016/bs.ircmb.2017.01.004. Epub 2017 Mar 7.  PMID: 28526136.

3: Cheffings TH, Burroughs NJ, Balasubramanian MK. Actomyosin Ring Formation and 
Tension Generation in Eukaryotic Cytokinesis. Curr Biol. 2016 Aug
8;26(15):R719-R737. doi: 10.1016/j.cub.2016.06.071. Review. PMID: 27505246.

4: Meitinger F, Palani S. Actomyosin ring driven cytokinesis in budding yeast.
Semin Cell Dev Biol. 2016 May;53:19-27. doi: 10.1016/j.semcdb.2016.01.043. Epub
2016 Feb 1. Review. PMID: 26845196

5: Skau CT, Waterman CM. Specification of Architecture and Function of Actin
Structures by Actin Nucleation Factors. Annu Rev Biophys. 2015;44:285-310. doi:
10.1146/annurev-biophys-060414-034308. Review.  PMID: 26098516.

**Comment by krchristie** [comment]
The Cheffings paper points to some potentially useful reviews and comparisons between organisms:

> Readers interested in a more in-depth discussion of actomyosin ring placement and/or formation are directed to a number of reviews that focus on S. pombe 2; 23 ;  24, S. cerevisiae 1; 20; 25 ;  26, and metazoan cells 5; 6 ;  27, and comparisons between these (and other) organisms have been discussed in other reviews 13; 28; 29 ;  30.

**Comment by ValWood** [comment]
Current def of actinomyosin ring
A cytoskeletal structure composed of actin filaments and myosin that forms beneath the plasma membrane of many cells, including animal cells and yeast cells, in a plane perpendicular to the axis of the spindle, i.e. the cell division plane. Ring contraction is associated with centripetal growth of the membrane that divides the cytoplasm of the two daughter cells. In animal cells, the contractile ring is located inside the plasma membrane at the location of the cleavage furrow. In budding fungal cells, e.g. mitotic S. cerevisiae cells, the contractile ring forms beneath the plasma membrane at the mother-bud neck before mitosis.

Although I don’t know how correct this is because I suspect the actin part of the actomyosin ring in animal cells is not “inside the plasma membrane” ( ? that would not be physically possible would it?????)
I think the organization is generally conserved, but I'm not sure whether the same names are used in other species.
Of the  47 components of the actinomyosin ring, at least 36 of them are identified in Metazoa, and we are actively looking for some of the small “missing” ones. 

I would suggest very simply

1. actinomyosin ring, proximal layer
The region of the actinomyosin ring where membrane bound scaffolds are located
2. actinomyosin ring, intermediate layer
The region of the actinomyosin ring in-between the proximal layer and the intermediate layer. This region contains the accessory protein network, some actin filaments and connections between the proximal layer and the distal layer
3. actinomyosin ring, distal layer
The region of the actinomyosin ring furthest from the plasma membrane. Contains actin filaments and cross linkers, myosin motors and connections to the plasma membrane through the distal layer.

I can confirm these proposed definitions with the author.
It would b great to get  these soon as we are launching the new PomBase next week, and we  hope to have th animation of this publication on the front page. 
http://preview.pombase.org/
http://preview.pombase.org/reference/PMID:28914606

**Comment by ValWood** [comment]
Metazoan arrangement seems broadly similar
http://jcs.biologists.org/content/joces/128/10/1877/F1.large.jpg

except there is a microtubule layer in addition .

This would make 
"actinomyosin ring, distal layer" a misleading name.
Maybe this could be a related synonymn
and this would be
actinomyosin ring, actin filament layer 

I think the defs are generic enough but I will check them with the authors.

**Comment by mah11** [comment]
>  I don’t know how correct this is because I suspect the actin part of the actomyosin ring in animal cells is not “inside the plasma membrane” 

I don't remember whether I worked on the GO:0005826 def back in the day, but reading it now I suspect this is simply dreadful wording intended to mean "just beneath the plasma membrane on the inside of the cell".

**Comment by ValWood** [comment]
OK, Kathy Gould says:
> Those GO terms you’ve developed are perfect for the different layers.

 and when I asked about the correspondence with metazoa

>We don’t know that definitively but probably because the pombe ring is built similarly to what has been found in other actin-based membrane structures in mammals but no one has delved into the ring.  MTs are not really present near the ring prior to constriction in mammals either and if they connect – it would probably be even more distal to the membrane via linking up with the actin layer.

so it is probably more similar than I thought from the above. The microtubules are the ones which link between the 2 cells (presumably the spindle?), which come into contact with the cytokinetic ring during abscission, not part of the contractile ring.

So we can probably use "distal actin filament layer' or similar?

**Comment by krchristie** [comment]
I can do this pretty quickly now that you've proposed specific terms with definitions, but have a couple remaining questions & comments, with  question 1 being the most important since that determines where I put the new terms:

1. How do the three new terms you would like relate to these two existing terms, -contractile ring' and 'actomyosin contractile ring'

- Are they supposed to be part of children like this:
-- contractile ring [GO:0070938]
--- actomyosin contractile ring [GO:0005826]
---- part_of actomyosin ring, proximal layer [GO:new]
---- part_of actinomyosin ring, intermediate layer [GO:new]
---- part_of actinomyosin ring, distal actin filament layer [GO:new]

- or is the existing term actomyosin contractile ring [GO:0005826] just meant to be the actin filament layer?


2. I'd like to add words to this definition to indicate that it's the layer next (aka proximal) to the plasma membrane. My suggestion is in bold, but feel free to edit:
**Term**: actinomyosin ring, proximal layer
**Def**: The region of the actinomyosin ring **adjacent to the plasma membrane** where membrane bound scaffolds are located 

3. Regarding the questioned wording in bold:

Current def of actinomyosin ring
A cytoskeletal structure composed of actin filaments and myosin that forms beneath the plasma membrane of many cells, including animal cells and yeast cells, in a plane perpendicular to the axis of the spindle, i.e. the cell division plane. Ring contraction is associated with centripetal growth of the membrane that divides the cytoplasm of the two daughter cells. **In animal cells, the contractile ring is located inside the plasma membrane at the location of the cleavage furrow**. In budding fungal cells, e.g. mitotic S. cerevisiae cells, the contractile ring forms beneath the plasma membrane at the mother-bud neck before mitosis.

I agree that this is meant to mean "located inside the cell adjacent to the plasma membrane". In fact, since I read the original wording to mean this, it took me awhile to even figure out what Val meant...

**Comment by ValWood** [comment]
Are they supposed to be part of children like this:
-- contractile ring [GO:0070938]
--- actomyosin contractile ring [GO:0005826]
---- part_of actomyosin ring, proximal layer [GO:new]
---- part_of actinomyosin ring, intermediate layer [GO:new]
---- part_of actinomyosin ring, distal actin filament layer [GO:new]

or is the existing term actomyosin contractile ring [GO:0005826] just meant to be the actin filament layer?

Yes like this. My understanding is that there are contractile rings (plants) which are not actomyosin based. We only use the actomyosin terms in f.y.

**Comment by ValWood** [comment]
Yes I agree to all of the definition changed. Gosh, I was reading the original definition *very* literally. I think I thought it was describing a difference. 


A cytoskeletal structure composed of actin filaments and myosin that forms beneath the plasma membrane of many cells, including animal cells and yeast cells, in a plane perpendicular to the axis of the spindle, i.e. the cell division plane. ~Ring contraction is associated with centripetal growth of the membrane that divides the cytoplasm of the two daughter cells.~ In animal cells, the contractile ring is located ~inside the plasma membrane at the location of~ at  the cleavage furrow. In budding fungal cells, e.g. mitotic S. cerevisiae cells, the contractile ring forms ~beneath the plasma membrane~ at the mother-bud neck before mitosis.

I would
i) remove the part describing ring contraction, this should only need to be in the ring contraction term?
ii) Don't need to mention plasma membrane at all in the 2nd part as it is the same as the first part, so it is totally redundant


(I'm not even sure that the differences between medially  dividing cells and budding are critical, but I guess they do no harm).

**Comment by krchristie** [comment]
Hi @ValWood 

Your terms are made as discussed above, but with these additional modifications listed below. I have also edited the definition of the existing term 'actomyosin contractile ring' as you suggested.

1. Note that you had proposed names using the phrase: '**actinomyosin ring**'. 
- part_of actinomyosin ring, proximal layer 
- part_of actinomyosin ring, intermediate layer 
- part_of actinomyosin ring, distal actin filament layer 

However our existing terms and the paper you cited in this ticket use the phrase '**actomyosin contractile** ring', so I have made the term names using the latter phrase.

2. I didn't put in the synonym of "actomyosin ring, distal layer" since it seemed like it could be confusing to have a synonym that was just generically distal. Of the several variant names that have been used in this ticket, "distal actin filament layer" seemed to be the clearest since it involved the word distal, but also specifies that it is an actin filament layer. At some point, we can add another term for an "actomyosin ring, distal microtubule layer" or something like that, and let the definition indicate the spatial relationship between the two distal layers in taxa where both occur.

3. While I had suggested that the existing term "GO:1903144 actomyosin contractile ring actin filament" could be made part_of the actomyosin contractile ring actin filament layer, but I realized that it shouldn't be since some of the actin filaments traverse the intermediate layer and anchor in the proximal layer. Thus, I thought that the existing relationship "part_of some actomyosin contractile ring" was accurate and sufficient.
